### ISSUE TITLE
[feat] command that shows mods/plugins with available updates

### ISSUE THREAD

**Issue by Sandr-aa** [ParadigmMC/mcman#69]
For example, having an `mcman check-updates` command, which either auto updates everything or prompts with an version selector like the one used when importing an modrinth mod over cli if an newer version has been released

**Comment by TheAlan404** [comment]
This will be added in `v2`, we'll also drop support for `"latest"` versions so using this `mcman update` command will be neccesary

**Comment by BANanaD3V** [comment]
> we'll also drop support for "latest" versions

Yes, this will make mcman reproducible, although we should have a strategy for when a plugin version is unavailable (say, author deleted the specific version)

**Comment by TheAlan404** [comment]
i dont think theres anything we can do in that situation other than failing the build
